Automattic/wp-calypso
# chore: remove danger

#### Changes proposed in this Pull Request

* Remove Danger.js

It is failing on OSS Citizen PRs because those PRs don't have access to an api token on circle which danger needs. We can revisit later if running Danger separate from CircleCI is an option but for now let's just unblock PRs from OSS Citizens.

#### Testing instructions

* Make sure there's no Danger job running on CircleCi
* Check if any Danger related files are removed from the repo
